pkg/patterns/declarative/application.go

```diff
@@ -0,0 +1,56 @@
+// application.go manages an Application[1]
+//
+// [1] https://github.com/kubernetes-sigs/application
+package declarative
+
+import (
+	"context"
+	"sort"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/kubebuilder-declarative-pattern/pkg/patterns/declarative/pkg/manifest"
+)
+
+func transformApplication(ctx context.Context, instance DeclarativeObject, objects *manifest.Objects, labelMaker LabelMaker) error {
+	log := log.Log
+
+	var app *manifest.Object
+	for _, o := range objects.Items {
```

Comment:
This looks similar to TransformApplicationFromStatus.  Should we extract a common function to "get the application from the manifest" (for follow-up PR).

Reply:
done!